src/loader/loader.c

```diff
@@ -470,6 +470,44 @@ post_analyze_hook(ParseState *pstate, Query *query, JumbleState *jstate)
 	}
 }
 
+/*
+ * Simplified check if a string is an UUID.
+ *
+ * This assumes that the UUID is a string consisting of dash-separated
+ * hexadecimal numbers and the string of of exact length 36 characters,
+ * containing 4 dashes, and with groups of digits of the size 8-4-4-4-12.
+ */
+static bool
+is_uuid(const char *str)
```

Comment:
Maybe it makes sense to reuse PostgreSQL types/functions for such validation

Reply:
I was thinking the same thing. Why not just call `uuid_in` on this string? If necessary, capture and rewrite the error.

Reply:
The function is `uuid_in`, which generate an error if it fails parsing, and capturing the error and replacing it with a different one was more complicated than writing this function. If you feel strongly about it, I can replace it.

Reply:
Switched to use `uuid_in`, as suggested.